src/search-scenario/azext_search_scenario/utils.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from azure.cli.core.style import print_styled_text, Style, is_modern_terminal
+
+
+def read_int(default_value=0):
+    ret = input()
+    if ret == '' or ret is None:
+        return default_value
+    while not ret.isnumeric():
+        ret = input("Please input a legal number: ")
```

Comment:
Use `valid` instead of `legal` as `legal` is related to the law.

Reply:
one